blocksuite/blocks/src/root-block/widgets/drag-handle/watchers/drag-event-watcher.ts

```diff
@@ -592,9 +592,13 @@ export class DragEventWatcher {
     this.widget.handleEvent('nativeDragStart', this._dragStartHandler, {
       global: true,
     });
-    this.widget.handleEvent('nativeDragMove', this._dragMoveHandler, {
-      global: true,
-    });
+    this.widget.handleEvent(
+      'nativeDragMove',
+      throttle(this._dragMoveHandler, DRAG_THROTTLE_TIME),
```

Comment:
Will making it async break the code?

Reply:
it will break following event handlers but luckily, we have only one event handler now

Reply:
maybe it is better to profile and short circuit event handlers when possible